index.md

```diff
@@ -1,41 +1,35 @@
 Introduction
 ============
 
-Welcome to the Sugar developers site. Here you will find all you need
-to get started and contribute to Sugar. You can also help us [improve
-this documentation](docs.md.html).
+Welcome to the Sugar developers site. 
+Here you will find all you need to get started and contribute to Sugar. 
+You can also help us [improve this documentation](docs.md.html).
 
 Getting started
 ---------------
 
-* [Setup a development environment](dev-environment.md.html)
 * [What can I do?](what-can-i-do.md.html)
-* [Write your own desktop activity](desktop-activity.md.html)
-* [Write your own web activity](web-activity.md.html)
 * [Contribute code](contributing.md.html)
-
-Release Notes
--------------
-* [See the Release Notes](release-notes.md.html)
+* [Release Notes](release-notes.md.html)
```

Comment:
This hasen't been updated for ages.  Maybe you could point it to http://wiki.sugarlabs.org/go/0.110